CONTRIBUTING.md

```diff
@@ -6,8 +6,8 @@
 
 ## Installation
 
-- Running `yarn` in the module's root directory will install everything you need for development.
-- Run `lerna bootstrap` in the module's root directory
+- Run `yarn` in the module's root directory to install everything you need for development.
+- Run `yarn build` in the root directory to build the modules and handle interlinking.
```

Comment:
Could you remove the handle interlinking bit since building doesn't do any linking, it just builds.